src/cmd/src/frontend.rs

```diff
@@ -217,6 +218,9 @@ impl StartCommand {
         logging::info!("Frontend start command: {:#?}", self);
         logging::info!("Frontend options: {:#?}", opts);
 
+        set_default_time_zone(opts.default_time_zone.as_deref().unwrap_or(""))
+            .context(InitTimeZoneSnafu)?;
```

Comment:
Since `set_default_time_zone` returns `Result`, how about process option value inside the function as well?
And, do we need to tolerant empty string case?